map/map_tests/kmz_unarchive_test.cpp

```diff
@@ -39,3 +40,13 @@ UNIT_TEST(KMZ_UnzipTest)
     TEST_EQUAL(bm.GetScale(), 0, ("KML wrong scale!"));
   }
 }
+
+UNIT_TEST(Multi_KML_KMZ_UnzipTest)
+{
+  std::string const kmzFile = GetPlatform().TestsDataPathForFile("kml_test_data/BACRNKMZ.kmz");
+  auto const filePaths = GetKMLOrGPXFilesPathsToLoad(kmzFile);
+  // Check that doc.kml (could be renamed to docN.kml) is skipped
+  TEST_EQUAL(5, filePaths.size(), ());
+  for (auto const & filePath : filePaths)
```

Comment:
If you already know what is in the test file, then compare all contained file names in the test.

Reply:
Fixed